network-store-server/src/main/resources/db/changelog/changesets/twt_regulationPoint_20241126T110000Z.sql

```diff
@@ -0,0 +1,66 @@
+-- two windings transformer
+insert into regulatingpoint (networkuuid, variantnum, regulatingequipmentid, regulatingequipmenttype, regulationmode, localterminalconnectableid, localterminalside, regulatingterminalconnectableid, regulatingterminalside)
+select networkuuid, variantnum, id, 'PHASE_TAP_CHANGER', phasetapchangerregulationmode, id, 'TWO', phasetapchangerterminalrefconnectableid, phasetapchangerterminalrefside
+from twowindingstransformer;
+insert into regulatingpoint (networkuuid, variantnum, regulatingequipmentid, regulatingequipmenttype, regulationmode, localterminalconnectableid, localterminalside, regulatingterminalconnectableid, regulatingterminalside)
+select networkuuid, variantnum, id, 'RATIO_TAP_CHANGER', ratiotapchangerregulationmode, id, 'TWO', ratiotapchangerterminalrefconnectableid, ratiotapchangerterminalrefside
+from twowindingstransformer;
+
+-- three windings transformer
+-- side 1
+insert into regulatingpoint (networkuuid, variantnum, regulatingequipmentid, regulatingequipmenttype, regulationmode, localterminalconnectableid, localterminalside, regulatingterminalconnectableid, regulatingterminalside)
+select networkuuid, variantnum, concat(id, '_ONE'), 'PHASE_TAP_CHANGER', phasetapchangerregulationmode1, id, 'ONE', phasetapchangerterminalrefconnectableid1, phasetapchangerterminalrefside1
+from threewindingstransformer;
+
+insert into regulatingpoint (networkuuid, variantnum, regulatingequipmentid, regulatingequipmenttype, regulationmode, localterminalconnectableid, localterminalside, regulatingterminalconnectableid, regulatingterminalside)
+select networkuuid, variantnum, concat(id, '_ONE'), 'RATIO_TAP_CHANGER', ratiotapchangerregulationmode1, id, 'THREE', ratiotapchangerterminalrefconnectableid1, ratiotapchangerterminalrefside1
+from threewindingstransformer;
```

Comment:
I'm not sure to understand how you chose the localterminalside: ONE, THREE, etc

Reply:
it is removed